internal/assert/assertions.go

```diff
@@ -0,0 +1,92 @@
+package assert
+
+// Testing check shortcuts
+
+import (
+	"fmt"
+	"reflect"
+	"runtime"
+	"strings"
+	"testing"
+)
+
+// Log message and fail if actualValue doesn't match expectedValue
+// *Note that DeepEqual will check the value pointed to by two pointers, ie don't use for checking
+//	pointer equality.
+func Equal(t *testing.T, expectedValue interface{}, actualValue interface{}, cm ...interface{}) bool {
```

Comment:
Can you provide an example of how `cm` should be used? It's not clear from the method signature.